### DISCUSSION TITLE
S2透视表的值数据单元格区域每个单元格能自定义聚合方式吗？

### DISCUSSION THREAD

**Discussion by tt0203** [antvis/S2#2848] [Q&A]
传统数据透视表一般有页筛选字段、行区域字段、列区域字段、值区域字段，
S2中可以在小计和合计中控制聚合方式，
然而我在使用如下数据源做透视表时发现，值区域数据每个单元格中应该是多个值的情况下，却只显示了多个值最后出现的值。
完整html代码如下：
`<head>
<script src="s2.min.js"></script>
</head>
<body>
<div id="test"></div>
<script>
 async function fx(){
 const tb=[ {年级:'初级',班级:'A',科目:'语文',成绩:100},{年级:'初级',班级:'A',科目:'数学',成绩:90},{年级:'初级',班级:'A',科目:'英语',成绩:80}, {年级:'初级',班级:'B',科目:'语文',成绩:95},{年级:'初级',班级:'B',科目:'数学',成绩:76},{年级:'初级',班级:'B',科目:'英语',成绩:82}, {年级:'初级',班级:'C',科目:'语文',成绩:93},{年级:'初级',班级:'C',科目:'数学',成绩:88},{年级:'初级',班级:'C',科目:'英语',成绩:86}, {年级:'中级',班级:'A',科目:'语文',成绩:75},{年级:'中级',班级:'A',科目:'数学',成绩:98},{年级:'中级',班级:'A',科目:'英语',成绩:87}, {年级:'中级',班级:'B',科目:'语文',成绩:95},{年级:'中级',班级:'B',科目:'数学',成绩:100},{年级:'中级',班级:'B',科目:'英语',成绩:70}, {年级:'中级',班级:'C',科目:'语文',成绩:90},{年级:'中级',班级:'C',科目:'数学',成绩:93},{年级:'中级',班级:'C',科目:'英语',成绩:84}, {年级:'高级',班级:'A',科目:'语文',成绩:88},{年级:'高级',班级:'A',科目:'数学',成绩:94},{年级:'高级',班级:'A',科目:'英语',成绩:80}, {年级:'高级',班级:'B',科目:'语文',成绩:70},{年级:'高级',班级:'B',科目:'数学',成绩:84},{年级:'高级',班级:'B',科目:'英语',成绩:86}, {年级:'高级',班级:'C',科目:'语文',成绩:90},{年级:'高级',班级:'C',科目:'数学',成绩:96}, ];
 const obop={width:600,height:480,hierarchyType: 'grid'};
 const dtop={fields:{rows:['年级'],columns:['班级'],values:['科目']},data:tb};
 const s2=new S2.PivotSheet('#test',dtop,obop);
 await s2.render();
 };
 fx();
 </script>
</body>`
测试情况：本地谷歌浏览器umd方式引入js文件。
下面截图中交叉位置单元格中按理说应该有多个科目值（数据源中初级A班有三个科目），为啥只显示最后一个值，有啥办法让值区域能自定义聚合，讲多个值全部显示出来，我查看了字段标记和数据格式化好像都没有控制这个的地方？

![QQ图片20240805152426](https://github.com/user-attachments/assets/a544a245-acbf-43c9-b80d-35b27aa11c63)

**Comment by lijinke666** [comment]
同 https://github.com/antvis/S2/discussions/2820

参考: https://github.com/antvis/S2/issues/2767#issuecomment-2164209195

**Reply by tt0203** [reply]
> 同 #2820
> 
> 参考: [#2767 (comment)](https://github.com/antvis/S2/issues/2767#issuecomment-2164209195)
我想你理解错我的意思了 ，我知道小计、合计怎么控制聚合，我现在的问题是每个数据单元格内也是需要聚合显示的！
你看我的数据源：初级A班有三个科目（语文、数学、英语三个科目），然而透视表值显示了最后一条记录值！值区域单元格内部不需要聚合处理吗？这是透视表最基本的能力啊？
![QQ图片20240807001746](https://github.com/user-attachments/assets/75713260-e13f-4b7a-9d55-e0d7c0888505)

**Reply by lijinke666** [reply]
不支持
